src/app/templates/about.tpl

```diff
@@ -19,16 +19,16 @@
         <div class="text-about">
             <div class="full-text">
                 <%= i18n.__("%s is the result of many developers and designers putting a bunch of APIs together to make the experience of watching torrent movies as simple as possible.", Settings.projectName) %><br/>
-                <%= i18n.__("We are an open source project. We are from all over the world. We love our movies. And boy, do we love popcorn.") %>
+                <%= i18n.__("We are an open source project. We are from all over the world. We love our movies. And boy, do we love our Popcorn.") %>
```

Comment:
I prefer the old text. I'm not sure how this line is supposed to be an improvement.